src/ucan.ts

```diff
@@ -115,7 +117,7 @@ export type Version = `${number}.${number}.${number}`
  */
 export interface JWTHeader {
   ucv: Version
-  alg: "EdDSA" | "RS256"
+  alg: string
```

Comment:
```suggestion
  alg: Alg
```

Should this be Alg instead ?